docs/examples/gce-ingress-controller/ingress-controller.yaml

```diff
@@ -0,0 +1,165 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: glbc
+  namespace: kube-system
+---
+apiVersion: rbac.authorization.k8s.io/
+kind: ClusterRole
+metadata:
+  name: system:controller:glbc
+rules:
+- apiGroups: [""]
+  resources: ["secrets", "endpoints", "services", "pods", "nodes", "namespaces", "configmaps", "events"]
+  verbs: ["describe", "get", "list", "watch", "update", "create", "patch"]
```

Comment:
I'm curious what the ingress controller updates or patches in the above resources. I would thing read-only access would be enough.